src/olympia/reviewers/templates/reviewers/addon_details_box.html

```diff
@@ -53,7 +53,7 @@ <h2>{{ addon_name }}</h2>
           </tr>
           <tr class="addon-status">
             <th>Status</th>
-            <td>
+            <td class="status-{{ amo.STATUS_CHOICES_API.get(addon.status, 'unknown') }}">
```

Comment:
Sort of a drive-by, but one of the complaints was that it wasn't always clear what happened to the add-on, so hopefully having a dedicated class (that applies a `color: ref`) for the disabled status will help a bit.